login-page/script.js

```diff
@@ -0,0 +1,19 @@
+const validUsers = {
+    'sarthak': '123',
+    // user2: 'password2'
+
+};
+
+document.getElementById('loginBtn').addEventListener('click', () => {
+    const username = document.getElementById('username').value;
+    const password = document.getElementById('password').value;
+
+    if(username==''||password==''){
```

Comment:
You can use === for password because it will help in type checking too.

Reply:
Ok sachin, I will correct it,